schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a web page context",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "gdpr",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+
+    "type": "object",
+    "properties": {
+        "basisForProcessing": {
+          "enum": ["consent", "contract", "legal obligation", "vital interest", "public task", "legitimate interests"],
+          "description": "GDPR basis for data collection & processing"
+        },
+        "documentId": {
+          "type": ["string", "null"],
+          "maxLength": 255,
+          "description": "ID for document detailing basis for processing"
+        },
+        "documentVersion": {
+          "type": ["string", "null"],
+          "maxLength": 5,
```

Comment:
Suggest making this longer e.g. 16 characters, so you can accommodate e.g. `12.4.2`

Reply:
Yup should've thought of that, fixing.